client/app/account/user/ssh/user-ssh.html

```diff
@@ -15,46 +15,45 @@ <h1 data-translate="user_ssh_title"></h1>
             <p>
                 <strong data-translate="user_ssh_key_explain_1"></strong>
             </p>
-            <p data-ng-bind-html="'user_ssh_key_explain_2' | translate:{ t0: ctrlSsh.guides.sshCreate }"></p>
+            <p>
+                <span data-translate="user_ssh_key_explain_2"></span>
+                <span data-translate="user_ssh_key_explain_2_guide"
+                      data-translate-values="{ t0: ctrlSsh.guides.sshCreate }"
+                      data-translate-compile></span>
```

Comment:
To allow tracking directive to use current scope and not `angular-translate`'s one